cmd/antrea-controller/options.go

```diff
@@ -63,6 +73,54 @@ func (o *Options) validate(args []string) error {
 	if len(args) != 0 {
 		return errors.New("no positional arguments are supported")
 	}
+
+	if o.config.EnableNodeIPAMController {
+		// Validate ClusterCIDRs
```

Comment:
could we move inside within the `if` block to its own `validateNodeIPAMControllerOptions` method?